integrationtest/org/daisy/dotify/formatter/test/resource-files/volume-break/volume-breaks-advanced4-input.obfl

```diff
@@ -1,7 +1,7 @@
 <obfl xmlns="http://www.daisy.org/ns/2011/obfl" version="2011-1" xml:lang="en">
 	<meta xmlns:dc="http://purl.org/dc/elements/1.1/">
-		<dc:title>Several break before volume test</dc:title>
-		<dc:description>Tests that break-before volume works when the number of volumes is increased.</dc:description>
+		<dc:title>???</dc:title>
```

Comment:
Please explain what is being tested here.

Reply:
I don't really know, that's the problem. Sometimes it's hard to figure out the original intention of a test. I removed the description that was there because it was obviously wrong. The test is a copy-paste of another test and the description was not modified.